apps/modernization-ui/src/apps/patient/profile/labReport/PatientLabReport.ts

```diff
@@ -0,0 +1,37 @@
+export enum Headers {
```

Comment:
this can be also exported as `Columns`

Reply:
For extra clarification, ES Lint will complain about the something being exported as `Headers` because there is already a global type named `Headers`.  Changing the name to `Columns` gets around that.

Reply:
This follows PatientInvestigation.ts example